airflow/utils/cli_app_builder.py

```diff
@@ -24,12 +24,15 @@
 from flask import Flask
 
 from airflow.www.extensions.init_appbuilder import AirflowAppBuilder, init_appbuilder
+from airflow.www.extensions.init_views import init_plugins
 
 
 @lru_cache(maxsize=None)
 def _return_appbuilder(app: Flask) -> AirflowAppBuilder:
     """Returns an appbuilder instance for the given app"""
-    return init_appbuilder(app)
+    init_appbuilder(app)
+    init_plugins(app)
```

Comment:
Doing it this way instead of just reverting sync-perm to using the full app saves us about .75 seconds. I wasn't able to reproduce the 3-5x gain in the original pr 🤷‍♂️.